Tasks/FtpUploadV1/Tests/L0.ts

```diff
@@ -1,16 +1,149 @@
 import fs = require('fs');
 import assert = require('assert');
 import path = require('path');
+import * as ttm from 'azure-pipelines-task-lib/mock-test';
 
 describe('FtpUploadV1 Suite', function () {
-    before(() => {
+    this.timeout(parseInt(process.env.TASK_TEST_TIMEOUT) || 20000);
+
+    it('check args: no serverEndpoint', (done: Mocha.Done) => {
+        this.timeout(parseInt(process.env.TASK_TEST_TIMEOUT) || 20000);
+
+        const tp = path.join(__dirname, 'L0NoServerEndpoint.js');
+        const tr: ttm.MockTestRunner = new ttm.MockTestRunner(tp);
+
+        tr.run();
+
+        assert(tr.stdOutContained('Input required: serverEndpoint'), 'Should have printed: Input required: serverEndpoint');
+        assert(tr.failed, 'task should have failed');
+
+        done();
+    });
+
+    it('check args: no rootFolder', (done: Mocha.Done) => {
+        this.timeout(parseInt(process.env.TASK_TEST_TIMEOUT || '') || 20000);
```

Comment:
Do we need to set it for every test or is it enough to set it in beginning of describe?

Reply:
Removed timeout from tests